doc/tfm/conf.py

```diff
@@ -25,7 +25,7 @@
 project = "Trusted Firmware-M"
 copyright = "2017-2021, ARM CE-OSS"
 author = "ARM CE-OSS"
-version = "1.7.0"
+version = "1.8.0"
```

Comment:
@shanthanordic/ @carlescufi / @gmarull  please confirm if we need the version updates for TF-M MCUboot etc.

Reply:
Yes TFM is 1.8.0 Version